### PR TITLE
Implement Flow statement

### PR DESCRIPTION
**Pull request by jwhb**
The `Flow` statement was missing, as it is not mentioned in the libnftables-json man page.

### CODE REVIEW

File: src/schema.rs [hunk 1]
```diff
@@ -298,16 +298,29 @@ pub struct Element {
 }
 
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+/// Flowtables allow you to accelerate packet forwarding in software (and in hardware if your NIC supports it) by using a conntrack-based network stack bypass.
```
Comment (JKRhb) [inline src/schema.rs:301]: This line could probably be split across multiple ones.

File: src/schema.rs [hunk 2]
```diff
@@ -298,16 +298,29 @@ pub struct Element {
...
     pub table: String,
+    /// Name the FlowTable is addressed by.
     pub name: String,
     #[serde(skip_serializing_if = "Option::is_none")]
+    /// Handle of the FlowTable object in the current ruleset.
     pub handle: Option<u32>,
+    /// Hook the FlowTable resides in.
     pub hook: Option<NfHook>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    /// The *priority* can be a signed integer or *filter* which stands for 0. Addition and subtraction can be used to set relative priority, e.g. filter + 5 equals to 5.
```
Comment (JKRhb) [inline src/schema.rs:315]: Small editorial suggestion:

```suggestion
    /// The *priority* can be a signed integer or *filter* which stands for 0.
    /// Addition and subtraction can be used to set relative priority, e.g., filter + 5 is equal to 5.
```

File: src/stmt.rs [hunk 3]
```diff
@@ -185,6 +188,15 @@ pub struct Limit {
     pub inv: Option<bool>,
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+/// Forward a packet to a different destination.
+pub struct Flow {
+    /// Operator on flow/set.
+    pub op: SetOp,
+    /// Operator on flow/set.
```
Comment (JKRhb) [inline src/stmt.rs:196]: Is this the correct doc comment here?

File: tests/json_tests.rs [hunk 4]
```diff
@@ -53,6 +53,69 @@ fn test_chain_table_rule_inet() {
...
+                        right: Expression::String("established".to_string()),
+                    }),
+                ],
+                handle: None,
+                index: None,
+                comment: None,
+            }))),
+        ],
+    };
+    let json = json!({"nftables":[{"table":{"family":"inet","name":"some_inet_table"}},{"flowtable":{"family":"inet","table":"some_inet_table","name":"flowed","hook":"ingress","prio":0,"dev":["lo"]}},{"chain":{"family":"inet","table":"some_inet_table","name":"some_inet_chain","type":"filter","hook":"forward","policy":"accept"}},{"rule":{"family":"inet","table":"some_inet_table","chain":"some_inet_chain","expr":[{"flow":{"op":"add","flowtable":"@flowed"}},{"match":{"left":{"ct":{"key":"state"}},"right":"established","op":"in"}}]}}]});
```
Comment (JKRhb) [inline tests/json_tests.rs:113]: Hmm, maybe here (as in other places) some linebreaking could be added?

**Review by JKRhb — COMMENTED** [review]
Changes look good to me, see some more minor comments below :)

**Comment by jwhb** [comment]
Applied all suggestions. Thanks for the review @JKRhb!

**Review by JKRhb — APPROVED** [review]
Awesome, LGTM :)

**Pull request merged**